### DISCUSSION TITLE
词汇的权重

### DISCUSSION THREAD

**Discussion by e71828** [KyleBing/rime-wubi86-jidian#107] [Ideas]
请问，当用户词汇的权重更大，部署后，能把用户后指定大权重的词提前么？

**Comment by e71828** [comment]
当然，是已经在主表中重复的词汇。

**Comment by e71828** [comment]
是不是要把个人 user.yaml 放在主表前面

**Comment by KyleBing** [comment]
https://github.com/KyleBing/rime-wubi86-jidian#%E5%85%AB%E8%AF%8D%E6%9D%A1%E6%9D%83%E9%87%8D

**Reply by e71828** [reply]
我可能没说清，我在子表中可以对主表中的字重新赋值更大的权重么

**Reply by KyleBing** [reply]
目前主表默认就是带初始权重的 10 20 30 40 50。子表字词可以通过修改自己的权重在调整在候选列表中的位置。
如果想调整主表中已有字词的顺序，就需要修改主表了。

https://github.com/KyleBing/rime-wubi86-jidian/blob/master/wubi86_jidian.dict.yaml

**Comment by e71828** [comment]
照这样，新的需求是给子表所有字加加权重offset，不过这是码表工具的活了
